toolchain/check/import_cpp.cpp

```diff
@@ -1139,28 +1138,21 @@ static auto MapQualifiedType(Context& context, SemIR::LocId loc_id,
 
   // TODO: Support other qualifiers.
   if (!quals.empty()) {
-    context.TODO(loc_id, llvm::formatv("Unsupported: qualified type: {0}",
-                                       type.getAsString()));
-    return {.inst_id = SemIR::ErrorInst::TypeInstId,
-            .type_id = SemIR::ErrorInst::TypeId};
+    return {.inst_id = SemIR::TypeInstId::None, .type_id = SemIR::TypeId::None};
   }
 
   return type_expr;
 }
 
 // Maps a C++ pointer type to a Carbon pointer type.
-static auto MapPointerType(Context& context, SemIR::LocId loc_id,
-                           clang::QualType type, TypeExpr pointee_type_expr)
-    -> TypeExpr {
+static auto MapPointerType(Context& context, clang::QualType type,
+                           TypeExpr pointee_type_expr) -> TypeExpr {
   CARBON_CHECK(type->isPointerType());
 
   if (auto nullability = type->getNullability();
       !nullability.has_value() ||
       *nullability != clang::NullabilityKind::NonNull) {
-    context.TODO(loc_id, llvm::formatv("Unsupported: nullable pointer: {0}",
-                                       type.getAsString()));
-    return {.inst_id = SemIR::ErrorInst::TypeInstId,
-            .type_id = SemIR::ErrorInst::TypeId};
+    return {.inst_id = SemIR::TypeInstId::None, .type_id = SemIR::TypeId::None};
```

Comment:
Maybe keep an explicit todo?

```suggestion
    // TODO: Support nullable pointers.
    return {.inst_id = SemIR::TypeInstId::None, .type_id = SemIR::TypeId::None};
```